src/plone/restapi/tests/test_content_patch.py

```diff
@@ -44,6 +44,18 @@ def test_patch_document(self):
         transaction.begin()
         self.assertEqual("Patched Document", self.portal.doc1.Title())
 
+    def test_patch_document_with_representation(self):
```

Comment:
@Gagaro would it be possible to add a few more tests here? What happens if you use "return=minimal", "return=", "", etc. Just to make sure we have all options covered and documented.

Reply:
Sorry, I was on holiday so I didn't see your message. I will be next week too, I don't know when I will be able to do that.